client/src/api/applicants.js

```diff
@@ -6,7 +6,7 @@ export const getAllApplicants = () => {
     });
 };
 export const getSkillsByApplicantId = id => {
-  return fetch(`/api/applicantSkills/${id}/skills`)
+  return fetch(`/api/applicantSkills/${id}`)
```

Comment:
This should be changed back because we are getting skills so will represent rest full API convention.